### PR TITLE
Update Proton VPN title

### PR DESCRIPTION
**Pull request by adamrusted**
**Issue:** contributes to #7451

**Similarweb rank:** [951](https://www.similarweb.com/website/proton.me/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [ ] I optimized the icon with SVGO or SVGOMG
  - [ ] The SVG `viewbox` is `0 0 24 24`

### Description
Changed PR to just the renaming of `ProtonVPN` to `Proton VPN`.

> **ℹ️ Note:**
> This update renames the icon from `ProtonVPN` to `Proton VPN` (adding a space). This should not cause an issue, as the slug removes the space anyway.

### CODE REVIEW

**Review by dirien — CHANGES_REQUESTED** [review]
Hi @adamrusted,

I see a difference in the overall shape/outline

image

**Comment by mondeja** [comment]
Hi @dirien. I'm moving this PR to the v10 milestone. If you and @adamrusted are capable of updating it before this Sunday, the change could be included in v9. In that case move it to the v9 milestone and merge when is ready. Thanks.

**Comment by adamrusted** [comment]
@mondeja and @dirien - I've updated this PR to just change the title of the icon from `ProtonVPN` to `Proton VPN`. Feel free to review and merge as/when appropriate :)

**Comment by dirien** [comment]
@adamrusted is this good to review and there is an issue with the `_data/simple-icons.json` file

File: sdk.mjs [hunk 1]
```diff
@@ -115,7 +115,7 @@ export const htmlFriendlyToTitle = (htmlFriendlyTitle) =>
     .replace(/&#([0-9]+);/g, (_, num) => String.fromCodePoint(parseInt(num)))
     .replace(
       /&(quot|amp|lt|gt);/g,
-      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
+      (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' })[ref],
```
Comment (mondeja) [inline sdk.mjs:118]: I think that this change was done by a previous version of Prettier. We commited it a few weeks ago and then failed again. Could you reformat?

**Review by mondeja — CHANGES_REQUESTED** [review]

**Review by PeterShaggyNoble — APPROVED** [review]

**Pull request merged**